game/hud/src/components/HUD/index.tsx

```diff
@@ -205,6 +224,9 @@ class HUD extends React.Component<HUDProps, HUDState> {
           zOrder={widget.position.zOrder}
           gridDivisions={10}
           locked={this.props.layout.locked}
+          selected={
+            this.state.selectedWidget && this.state.selectedWidget.name === type ? true : false
```

Comment:
You don't even need a ternary operator for this guy.

Just do the
`this.state.selectedWidget && this.state.selectedWidget.name === type` and it will come out as a boolean.